.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+
+install:
+  - export PATH=/tmp/grpc-deps/bin:$PATH
```

Comment:
It seems like we don't need LD_LIBRARY_PATH if we're running protoc out of the directory it was installed into by configure --prefix, it searches for ../lib apparently